python/tk_desktop/project_model.py

```diff
@@ -19,6 +19,7 @@
 
 shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
 shotgun_login = sgtk.platform.import_framework("tk-framework-login", "shotgun_login")
+shotgun_api3 = sgtk.platform.import_framework("tk-framework-login", "shotgun_api3")
```

Comment:
That's what I did at first. Unfortunately, the exception being thrown is from tk-framework-login's version of shotgun_api3. Therefore, if I catch the Fault exception from tank_vendor.shotgun_api3, it passes right through... That's why I am stuck catching the one from tk-framework-login.shotgun_api3.